lib/src/components/Stepper/stepper.types.ts

```diff
@@ -0,0 +1,55 @@
+import { MutableRefObject, ReactNode } from "react";
+import { DividerProps } from "../Divider";
+
+export type TStepperSizes = "xs" | "sm" | "md" | "lg" | "xl";
+
+export type TStepperBorderRadius = "xs" | "sm" | "md" | "lg" | "xl";
+
+export type TStepperProps = JSX.IntrinsicElements["div"] & {
+  activeStep: number;
+  children: ReactNode;
+  onStepClick?: (stepIndex: number) => void;
+  orientation?: "horizontal" | "vertical";
+  labelOrientation?: "horizontal" | "vertical";
+  size?: string;
+  borderRadius?: string;
+  baseStyles?: Partial<TStepperTheme[1]["baseStyles"]>;
+  completedStyles?: Partial<TStepperTheme[1]["completedStyles"]>;
+  progressStyles?: Partial<TStepperTheme[1]["progressStyles"]>;
+  ref?: MutableRefObject<HTMLDivElement | null>;
+  icon?: ReactNode;
+  completedIcon?: ReactNode;
+  progressIcon?: ReactNode;
+  isLastChild?: boolean;
+  stepState?: "stepProgress" | "stepCompleted" | "stepInactive";
+  dividerProps?: DividerProps;
+};
+
+export type TStepperStepProps = JSX.IntrinsicElements["div"] &
+  Omit<TStepperProps, "activeStep" | "children"> & {
+    children?: ReactNode;
+    ref?: MutableRefObject<HTMLDivElement | null>;
+  };
+
+export type TStepperTheme = [
```

Comment:
why use `T` here? `I` meaning interface should be used

Reply:
My bad, will Update the same

Reply:
ahhh okok we are using type here, until now we use `I` regardless to seperate variables from types

Reply:
lets change it to`I` it think having both `T` and `I` will cause confusions in future